src/eei.h

```diff
@@ -116,8 +116,28 @@ struct EthereumInterface : ShellExternalInterface {
 };
 
 struct GasSchedule {
+  static constexpr unsigned storageLoad = 200;
   static constexpr unsigned storageStoreCreate = 20000;
   static constexpr unsigned storageStoreChange = 5000;
+  static constexpr unsigned log = 375;
+  static constexpr unsigned logData = 8;
+  static constexpr unsigned logTopic = 375;
+  static constexpr unsigned create = 32000;
+  static constexpr unsigned call = 700;
+  static constexpr unsigned copy = 3;
+  static constexpr unsigned blockhash = 800;
+  static constexpr unsigned balance = 400;
+  static constexpr unsigned stepGas0 = 0;
```

Comment:
Seems unused.

Reply:
removed now. I kept it there for the sake of correspondence to EVM gas schedule, but this is now unneeded